### PR TITLE
Mysql in dbconfig xml

### PR DESCRIPTION
**Pull request by mmolinac**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions
* Remember to set **idealista:develop** as base branch;

### Description of the Change

I've noticed that the default behaviour is not to configure a database connection for JIRA.
Even in the case you provided `oracle` as the selected type, you have to fill in several other parameters.

I needed to configure a MySQL database and that case wasn't being taken into account.

I fixed the `dbconfig.xml.j2` template as well as the required steps to install a valid MySQL JDBC connector.

I also put the connector in the `lib/` folder of the JIRA installation and now you can have a working installation of JIRA with a MySQL database.

I've also performed the changes required by @dortegau in the closed PR https://github.com/idealista/jira-role/pull/7

### Benefits

You'll be able to configure a JIRA installation with a database other than Oracle / Oracle10g.
I've only filled in the `mysql` case, but you can add others too in the future, provided that you also add that required part of code as well.

### Possible Drawbacks

From the code base provided, if you use the default configuration there's no possible drawback.
If you fill in a configuration for other DB than `oracle` / `oracle10g`, you must provide additional parameters, of course.

### Applicable Issues

Solves https://github.com/idealista/jira-role/issues/4

### CODE REVIEW

**Comment by mmolinac** [comment]
Hmm. It seems that there's another problem with `java-role` during the tests.

**Comment by dortegau** [comment]
Hi @mmolinac, 

You're trying to tests this role using 2.0.2 version of Java Role as dependency that is outdated (Oracle removed those deprecated versions and aren't available anymore). Could you please pull latests changes from develop to use [version 3.3.0](https://github.com/idealista/jira-role/blob/develop/tests/requirements.yml) instead?

File: defaults/main.yml [hunk 1]
```diff
@@ -7,6 +7,8 @@ jira_download_dir: /jiradownload
 jira_installation_dir: /opt/atlassian/jira
 jira_attachments_dir: "{{ jira_home }}/data/attachments"
 jira_version: 7.9.2
+jira_mysql_repo_url: 'http://repo.mysql.com/apt/debian/'
```
Comment (jnogol) [inline defaults/main.yml]: These make more sense in `vars/mysql-vars.yml`, IMO

Comment (mmolinac) [reply defaults/main.yml]: Yes, I just realized that.
Done.

**Comment by mmolinac** [comment]
I'm a bit confused by the error reported by the test suite:

`"failure 1 during daemon-reload: Failed to reload daemon: Connection timed out\n"`

It seems that `systemd` is failing to perform a `daemon-reload`

Has this happened to you before? Could you check which version of `systemd` the test host has?
Because is neither that the startup script has changed, nor that the start step is even reached.

**Comment by jnogol** [comment]
Yeah, don't worry about it. It's some sort of timeout that occurs from time to time in CircleCI/TravisCI. Sometimes retrying several times do the work, but meh, I'll test it out tomorrow in my computer and if it works, ok with the merge.

**Comment by mmolinac** [comment]
@jnogol I pushed another change to trigger the tests and see if I have luck this time, but as you said, it failed again.

**Comment by jnogol** [comment]
Yep, it's pretty annoying sometimes. Don't worry, I've tried in my local and everything works, so ok for me

**Review by jnogol — APPROVED** [review]

**Review by dortegau — APPROVED** [review]

**Pull request merged**